Source/LinqToDB.EntityFrameworkCore/EFCoreMetadataReader.cs

```diff
@@ -278,7 +277,7 @@ static DataType DbTypeToDataType(DbType dbType)
 						{
 							(T)(Attribute)new ColumnAttribute
 							{
-								Name            = prop.GetColumnName(storeObjectId!.Value),
+								Name            = storeObjectId != null ? prop.GetColumnName(storeObjectId.Value) : null,

```

Comment:
storeObjectId is not value type anymore?

Reply:
It is, but there were assumption that GetStoreObjectIdentifier will return non-null value. I'm not sure it is always true, so I've replaced code a bit to treat it as nullable value to avoid NRE